src/page/product-added-modal.page.ts

```diff
@@ -4,7 +4,7 @@ export class ProductAddedModalPage {
   private proceedToCheckoutButton: ElementFinder;
 
   constructor () {
-    this.proceedToCheckoutButton = $('[style*="display: block;"] .button-container > a');
+    this.proceedToCheckoutButton = $('.button-container > .button-medium > span');
```

Comment:
try to find a better locator, something that uses the name or title attribute for example, in that way, we can understand what element you are referring